dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/citation/CitationData.java

```diff
@@ -25,7 +25,9 @@
     private String year;
     private Date date;
     private String releaseYear;
-    private GlobalId persistentId;
+    private GlobalId persistentId; // this field can contain dataset or file pid depending on context
```

Comment:
Maybe this comment should be above getter?

Reply:
I think you can remove it now since you added it above getter :P

Reply:
Yes, but I thought that's also good to leave it here, as not everyone would expect such information on getter.
But I can delete it if you want.

Reply:
Hmm, In our projects I expect info above getters, since we have a rule that in entities javadocs are on getters. So for me info here is just duplication.